### PR TITLE
Add pamela deployment

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: resources/console/charts/pamela/values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  annotations: {}
+serviceAccountName: apiserver-proxy # !!!
+istio:
+  namespace: istio-system
+verifySSL: false
+tracing:
+  debug: false
+  serviceSpanName: console-pamela
+
+systemNamespaces: "istio-system,knative-eventing,kube-public,kube-system,kyma-installer,kyma-integration,kyma-system,natss,compass-system,kube-node-lease,kubernetes-dashboard"
```
Comment (dariadomagala) [inline resources/console/charts/pamela/values.yaml:23]: we don't need that

File: resources/console/charts/web/values.yaml [hunk 2]
```diff
@@ -35,6 +35,7 @@ core_ui:
   resources: {}
 managementPlane: {}
 replicaCount: 1
+resources: {}
```
Comment (dariadomagala) [inline resources/console/charts/web/values.yaml:38]: resources are made for console and core-ui separately (lines 25 & 35)

File: resources/console/charts/pamela/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        image: "{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+          - name: jwksUri
+            value: "https://dex.{{ .Values.global.ingress.domainName }}/keys"
+        ports:
+        - containerPort: {{ .Values.service.internalPort }}
+          name: {{ .Values.service.internalPortName }}
+          protocol: TCP
+        resources:
```
Comment (dariadomagala) [inline resources/console/charts/pamela/templates/deployment.yaml:48]: we should set limits and requests for evaluation/production profiles (can be done as a followup)

File: resources/console/charts/pamela/templates/deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        heritage: "{{ .Release.Service }}"
+        kyma-project.io/component: backend
+    spec:
+      serviceAccountName: apiserver-proxy
+      {{ if .Values.global.isLocalEnv }}
+      hostAliases:
+        - ip: {{ .Values.global.minikubeIP }}
+          hostnames:
+            - "dex.{{ .Values.global.ingress.domainName }}"
+            - "minio.{{ .Values.global.ingress.domainName }}"
```
Comment (dariadomagala) [inline resources/console/charts/pamela/templates/deployment.yaml:35]: do we need minio in pamela?

File: resources/console/charts/pamela/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    rollingUpdate:
+      maxUnavailable: 0
+  template:
+    metadata:
+      labels:
+        app: {{ template "name" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
+        kyma-project.io/component: backend
```
Comment (dariadomagala) [inline resources/console/charts/pamela/templates/deployment.yaml:27]: `pamela`?

**Review by dariadomagala — CHANGES_REQUESTED** [review]

File: resources/console/charts/web/values.yaml (hunk 2 above)
Comment (parostatkiem) [reply resources/console/charts/web/values.yaml:38]: something went wrong at merge 🤦

File: resources/console/charts/pamela/templates/deployment.yaml (hunk 3 above)
Comment (parostatkiem) [reply resources/console/charts/pamela/templates/deployment.yaml:48]: Yes yes, followup!

File: resources/console/charts/pamela/templates/deployment.yaml (hunk 5 above)
Comment (parostatkiem) [reply resources/console/charts/pamela/templates/deployment.yaml:27]: No, backend. It's a label for Grafana (? I think) which can only have one of the following values `frontend`, `backend`, `controller`.

File: resources/console/charts/pamela/templates/deployment.yaml [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+        - containerPort: {{ .Values.service.internalPort }}
+          name: {{ .Values.service.internalPortName }}
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 300m
+            memory: 300Mi
+          requests:
+            cpu: 80m
+            memory: 180Mi
```
Comment (dariadomagala) [inline resources/console/charts/pamela/templates/deployment.yaml:47]: do we actually need those values to be so high?

Comment (parostatkiem) [reply resources/console/charts/pamela/templates/deployment.yaml:47]: I don't think so. That's why we have a follow-up task on our board 😄

**Review by dariadomagala — APPROVED** [review]

**Pull request merged**